packages/core/minos-microservice-networks/minos/networks/decorators/definitions/http/abc.py

```diff
@@ -12,7 +12,8 @@
 class HttpEnrouteDecorator(EnrouteDecorator):
     """Http Enroute Decorator class."""
 
-    def __init__(self, path: Optional[str] = None, method: Optional[str] = None, url: Optional[str] = None):
+    def __init__(self, path: Optional[str] = None, method: Optional[str] = None, url: Optional[str] = None, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Could you add `kwargs` propagation to the remaining `EnrouteDecorator` heirs, like `BrokerEnrouteDecorator`, `PeriodicEnrouteDecorator`, `GraphQlEnrouteDecorator` and so on?